src/styles/card.scss

```diff
@@ -0,0 +1,118 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+
+  padding: 32px 16px 16px;
+
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  background: $button-color-hover;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-accent;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    margin-top: 0;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__count-review {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    margin-bottom: 0;
+    width: 53px;
+    height: 16px;
+    margin-top: 0;
+    font-size: 10px;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    margin-top: 0;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main-accent;
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 164px;
+    height: 38px;
+
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    color: $button-color-hover;
+
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    background-color: $button-color;
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    &:hover {
+      background-color: $button-color-hover;
+      color: $button-color;
+      border-color: $button-color;
+    }
+  }
+  &:hover {
+    transform: scale(1.2);
+    .card__title {
+      color: $color-title-new;
+    }
+
+    .nav__link {
+      color: $button-color;
+    }
```

Comment:
The selector '.nav__link' inside the '.card:hover' block seems unrelated to the card component. Ensure that this selector is intended to be styled within the card hover state, or consider removing it if it's not relevant.